vmdb/app/models/mixins/authentication_mixin.rb

```diff
@@ -74,8 +94,20 @@ def update_authentication(data, options = {})
     data.each_pair do |type, value|
       cred = self.authentication_type(type)
       current = {:new => nil, :old => nil}
-      current[:new] = {:user => value[:userid], :password => value[:password]} unless value[:userid].blank?
-      current[:old] = {:user => cred.userid, :password => cred.password} if cred
+
+      if value[:auth_key]
+        # TODO(lsmola) figure out if there is a better way. Password field is replacing \n with \s, I need to replace
```

Comment:
Can you clarify this comment... by "Password field" do you mean the database field or the UI field or something else?